pkg/aversiveplusplus/Makefile.include

```diff
@@ -0,0 +1,20 @@
+MODULES = $(BINDIRBASE)/pkg/$(BOARD)/aversiveplusplus/modules
```

Comment:
I think it's safer to name that variable `AVERSIVEPLUSPLUS_MODULES` or something like this.